src/rules/variableNameRule.ts

```diff
@@ -64,11 +87,19 @@ class VariableNameWalker extends Lint.RuleWalker {
         }
     }
 
-    private handleVariableName(name: ts.Identifier) {
+    private handleVariableNameFormat(name: ts.Identifier) {
```

Comment:
no need to have separate methods, just keep the old method name and have 2 `if` blocks, that'd simplify the code so much